docs/design/2021-09-13-parser-as-submodule-of-tidb.md

```diff
@@ -0,0 +1,87 @@
+# Proposal: Move parser back to pingcap/tidb
+
+- Author(s): [xhebox](http://github.com/xhebox)
+- Discussion: [Move parser back to pingcap/tidb](https://internals.tidb.io/t/topic/385)
+- Tracking Issue: None for now.
```

Comment:
@xhebox please create an tracking issue before merging this RFC.

Reply:
I won't merge it now, since I've promised to have 3 LGTM and wait for 3 days. I would create one later, you could give the 3rd LGTM.

Reply:
OK. I'll review this proposal today.

Reply:
#28257, here it is.